app/src/org/odk/collect/android/activities/FormEntryActivity.java

```diff
@@ -2762,6 +2763,10 @@ public void savingComplete(int saveStatus) {
      * @return status as determined in FormEntryController
      */
     public int saveAnswer(IAnswerData answer, FormIndex index, boolean evaluateConstraints) {
+
+        ACRAUtil.addCustomData(ACRAUtil.LAST_ANSWER_DATA, (answer == null ? "null" : answer.getDisplayText()));
```

Comment:
This is a super interesting idea. I like this a lot. For debugging it might be good to get both the display and data text in case the data has something interesting going on.

Also: Would be good to make sure every wrapping call like this is 100% execution safe. It's good to check the null thing here, but would like it to be the case that it's impossible for our logging calls to crash things if at all possible (not sure how cleanly we can wrap those contexts in java, though)